torchvision/io/video_reader.py

```diff
@@ -154,6 +206,17 @@ def get_metadata(self) -> Dict[str, Any]:
         Returns:
             (dict): dictionary containing duration and frame rate for every stream
         """
+        if self.backend == "pyav":
+            metadata = {}  # type:  Dict[str, Any]
+            for stream in self.container.streams:
+                if stream.type not in metadata:
+                    metadata[stream.type] = {"fps": [], "duration": []}
+
+                rate = stream.average_rate if stream.average_rate is not None else stream.sample_rate
+
+                metadata[stream.type]["duration"].append(float(stream.duration * stream.time_base))
+                metadata[stream.type]["fps"].append(float(rate))
```

Comment:
for audio we use framerate instead of fps, I think this is the reason build_docs is failing